src/client/datascience/data-viewing/dataViewer.ts

```diff
@@ -140,23 +163,35 @@ export class DataViewer extends WebviewPanelHost<IDataViewerMapping> implements
         return output;
     }
 
-    private async getAllRows() {
+    // Deprecate this
+    private async getAllRows(sliceExpression?: string) {
```

Comment:
As part of moving all the debugger row chunking logic to the UI, I want to suggest that we deprecate usage of `getAllRows`, because we can express the same thing with `getRowChunk` (just ask for all the rows in the chunk request). But I wanted to leave this for the review in case I'm missing a reason to have this here.

Reply:
Makes sense to me. It's a full subset of the other command.